README.md

```diff
@@ -266,10 +266,15 @@ Another example is the Node.js [unofficial-builds project](https://github.com/no
 
     export N_NODE_MIRROR=https://unofficial-builds.nodejs.org/download/release
 
-You may need to specify the architecture explicitly if not autodetected by `n`, such as using `musl` `libc` on Alpine:
+You may need to specify the architecture explicitly if not autodetected by `n`, such as using `musl` `libc` on Alpine. You can do that with using `N_ARCH`:
 
     export N_NODE_MIRROR=https://unofficial-builds.nodejs.org/download/release
+    export N_ARCH=x64-musl
     apk add bash curl libstdc++
+    n install lts
+
+or using `--arch`:
+
     n --arch x64-musl install lts
```

Comment:
Can leave out the `--arch` usage here.

Reply:
(Left a comment at the end of the line, to avoid the illusion that both are needed...)